composer.lock

```diff
@@ -7662,22 +7670,27 @@
         },
         {
             "name": "psr/container",
-            "version": "1.1.2",
+            "version": "2.0.2",
```

Comment:
Hmm, upgrading this dependency to a new major could lead to breaking changes no? Like method prototype for example, and this dependency is not used in tests only but pretty much everywhere we use containers